eZ/Bundle/EzPublishCoreBundle/Resources/views/default/embed.html.twig

```diff
@@ -0,0 +1,3 @@
+{% for field in content.FieldsByLanguage %}
+    {{ ez_render_field(content, field.fieldDefIdentifier) }}
+{% endfor %}
```

Comment:
yes sir :). thing is don't really know which html tags should i use to enclose titles or if we plan to add more div wrappers and classes... is there any kind of guide for that?